frontend/src/app/features/folios/redux/FolioSlice.ts

```diff
@@ -183,7 +183,12 @@ const folioSlice = createSlice({
         state.fetchRequestStatus = RequestStatus.failed;
       })
       .addCase(addFolioItem.fulfilled, (state, action) => {
-        if (action?.payload?.data?.addFolioItem?.httpStatusCode === 400)
+        if (
+          action?.payload?.data?.addFolioItem?.httpStatusCode === 400 ||
```

Comment:
can we use the http code instead of the numeric value

Reply:
Added status codes to a const and referred it.